jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -134,6 +134,16 @@ def init_decoding(self, event={}):
         dq_layer = self.get_dq_layer()
         dq_layer.composite._allow_bad_alpha = True
 
+        # for cubeviz, also change uncert-viewer defaults to
+        # map the out-of-bounds regions to the cmap's `bad` color:
+        if self.app.config == 'cubeviz':
+            uncert_viewer = self.app.get_viewer(
+                self.app._jdaviz_helper._default_uncert_viewer_reference_name
+            )
+            for layer in uncert_viewer.layers:
+                layer.composite._allow_bad_alpha = True
+                layer.force_update()
```

Comment:
https://github.com/spacetelescope/jdaviz/assets/42986583/f1d9a6e5-507e-43a1-8216-b0333973d6da


I think this is a result of the background preferences, and it may be a null point. The relative opacity of the DQ layer does make the SCI data less visualizable compared to the previous commits and having the less desirable background. I'm unsure of the usability requirements when someone is focused on looking at DQ and if they would be okay toggling the layer on and off through the data menu.

Reply:
Thanks for these points. The DQ information is only useful if you can flip between science and DQ arrays, and see which science data may be affected by artifacts. I agree that it can be visually confusing to have both DQ and SCI open at the same, so we have the opacity slider and visibility in the data menu.

Reply:
Is the "darkening" of the image due to the DQ layer "fixed" by the upstream changes to glue?  Or does that only affect outside the bounds of the DQ layer?